BEIMA.Backend.FT/BuildingFT.cs

```diff
@@ -200,5 +200,57 @@ public async Task BuildingInDatabase_DeleteBuilding_BuildingDeletedSuccessfully(
             var ex = Assert.ThrowsAsync<BeimaException>(async () => await TestClient.GetBuilding(buildingId));
             Assert.That(ex?.StatusCode, Is.EqualTo(HttpStatusCode.NotFound));
         }
+
+        [Test]
+        public async Task BuildingInDatabase_UpdateBuilding_ReturnsUpdatedBuilding()
+        {
+            // ARRANGE
+            var origBuilding = new Building
+            {
+                Name = "Student Union",
+                Number = "1234",
+                Notes = "Some building notes.",
+                Location = new Location
+                {
+                    Latitude = "123.456",
```

Comment:
The lat values are out of range, we should probably make those valid. Can't remember if we are planning to add lat/long validation to the backend, but would be good to have valid test values if we do

Reply:
Forgot about this again, yeah we should have lat/lon validation in by the end of the sprint